README.md

```diff
@@ -52,8 +52,8 @@ require CUDA >= 6.0. Additionally, the _sanity_ test requires check >= 0.9.8 and
 subunit.
 
 ```shell
-# On RHEL
-$ sudo yum install check check-devel subunit subunit-devel
+# On RHEL (epel-release repository might be required)
```

Comment:
I would say epel-release is required, not might be.

Reply:
We need dkms, which usually be found in epel-release. So that repo is not actually required if users have already installed dkms. But let me make the message easier to understand.